packages/grid/_modules_/grid/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -54,6 +37,70 @@ export const cleanFilterItem = (item: GridFilterItem, apiRef: GridApiRef) => {
   return cleanItem;
 };
 
+const filterModelDisableMultiColumnsFilteringWarning = buildWarning([
+  'MUI: The `filterModel` can only contain a single item when `prop.disableMultipleColumnsFiltering` is set to `true`.',
+  'If you are using the `DataGrid` community version, this property is always `true`.',
+]);
+
+const filterModelMissingItemIdWarning = buildWarning(
+  "MUI: The 'id' field is required on `filterModel.items` when you use multiple filters.",
+);
+
+export const sanitizeFilterModel = (
+  model: GridFilterModel,
+  disableMultipleColumnsFiltering: boolean,
+  apiRef: GridApiRef,
+) => {
+  if (model.items.length > 1) {
```

Comment:
We're not setting the default `operatorValue` when `filterModel` contains only 1 item. Actually this causes a bug which could be solved here: https://codesandbox.io/s/controlledfilters-material-demo-forked-u1utu?file=/demo.tsx

![image](https://user-images.githubusercontent.com/42154031/153610337-dc489824-1e28-4170-8659-4a3ba59f1cd3.png)

Reply:
Indeed, fixed